contracts/internals/controllable.sol

```diff
@@ -25,12 +25,8 @@ import "../externals/initializable.sol";
 
 /// @title Controllable implements access control functionality of the Controller found via ENS.
 contract Controllable is ENSResolvable {
-    // Default values for mainnet ENS
-    // controller.tokencard.eth
-    bytes32 private constant _DEFAULT_CONTROLLER_NODE = 0x7f2ce995617d2816b426c5c8698c5ec2952f7a34bb10f38326f74933d5893697;
-
-    /// @dev Is the registered ENS node identifying the controller contract.
-    bytes32 private _controllerNode = _DEFAULT_CONTROLLER_NODE;
+    /// @dev Registered ENS node identifying the controller contract (default: controller.tokencard.eth).
+    bytes32 private _controllerNode = 0x7f2ce995617d2816b426c5c8698c5ec2952f7a34bb10f38326f74933d5893697;
```

Comment:
It's the same thing logically but it costs more to also store the DEFAULT constant - I think Yannis recently added defaults this way in the ensResolvable.sol so I changed it here also. IMO this way is better and we should do away with the DEFAULT pattern since this works the same way and costs less to deploy.

Reply:
Yes, I get it and it is nicer ... 

`contracts/internals/tokenWhitelistable.sol:    bytes32 private constant _DEFAULT_TOKEN_WHITELIST_NODE = 0xe84f90570f13fe09f288f2411ff9cf50da611ed0c7db7f73d48053ffc974d396;
contracts/internals/tokenWhitelistable.sol:    bytes32 private _tokenWhitelistNode = _DEFAULT_TOKEN_WHITELIST_NODE;
contracts/walletCache.sol:    bytes32 private constant _DEFAULT_LICENCE_NODE = 0xd0ff8bd67f6e25e4e4b010df582a36a0ee9b78e49afe6cc1cff5dd5a83040330;
contracts/walletCache.sol:    bytes32 private constant _DEFAULT_TOKEN_WHITELIST_NODE = 0xe84f90570f13fe09f288f2411ff9cf50da611ed0c7db7f73d48053ffc974d396;
contracts/walletCache.sol:    bytes32 private constant _DEFAULT_WALLET_DEPLOYER_NODE = 0x1d0c0adbe6addd93659446311e0767a56b67d41ef38f0cb66dcf7560d28a5a38;
contracts/walletCache.sol:    bytes32 public licenceNode = _DEFAULT_LICENCE_NODE;
contracts/walletCache.sol:    bytes32 public tokenWhitelistNode = _DEFAULT_TOKEN_WHITELIST_NODE;
contracts/walletCache.sol:    bytes32 public walletDeployerNode = _DEFAULT_WALLET_DEPLOYER_NODE;
contracts/walletDeployer.sol:    bytes32 private constant _DEFAULT_WALLET_CACHE_NODE = 0xaf553cb0d77690819f9d6fbaa04416e1fdcfa01b2a9a833c7a11e6ae0bc1be88;
contracts/walletDeployer.sol:    bytes32 public walletCacheNode = _DEFAULT_WALLET_CACHE_NODE;
tools/echidna/internals/controllable.sol:    bytes32 constant _DEFAULT_CONTROLLER_NODE = 0x7f2ce995617d2816b426c5c8698c5ec2952f7a34bb10f38326f74933d5893697;
tools/echidna/internals/controllable.sol:        return controllerNode() == _DEFAULT_CONTROLLER_NODE;`

It looks like we should be changing this everywhere then.

Reply:
The constant value is replaced at compile time (a getter is added if it is public), In theory it shouldn't add any overhead. We just have to pick a pattern and stick to it :)

Reply:
Yes, i thought so too, that is what other languages would do, but I didn't want to say for certain that Solidity. We should revert this, and to keep the pattern, as was.

Reply:
I reverted it but we don't really have a pattern in place we use both ways in different contracts.